muchopper/web/scss/app.scss

```diff
@@ -116,6 +116,7 @@ header#main.slim > .searchbox {
 	flex: 1 1 auto;
 	display: flex;
 	flex-direction: column;
+	margin: auto;
```

Comment:
On a techincal note, the proper way to achieve this would be ``align-self: center;``, since this is a flex item.